parm/fv3lam_rrfs.xml

```diff
@@ -2,7 +2,7 @@
 <postxml xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="EMC_POST_CTRL_Schema.xsd">
 <!--  Lin Gan:  20150427 validated -->
     <paramset>
-    <datset>PRSLEV</datset>
+    <datset>BGDAWP</datset>
```

Comment:
PRSLEV is the same used for operation. Should not change this.
The script will make the file name change for generating BGDAWP for now. But we will get rid of this file name when downstream systems are ready.

Reply:
OK.  Sorry, I will revert this.